panther_controller/launch/controller.launch.py

```diff
@@ -34,31 +34,15 @@
 
 
 def generate_launch_description():
-    panther_version = LaunchConfiguration("panther_version")
-    declare_panther_version_arg = DeclareLaunchArgument(
-        "panther_version",
-    )
-
-    use_sim = LaunchConfiguration("use_sim")
-    declare_use_sim_arg = DeclareLaunchArgument(
-        "use_sim",
-        default_value="False",
-        description="Whether simulation is used",
-    )
-
-    wheel_config_path = LaunchConfiguration("wheel_config_path")
-    declare_wheel_config_path_arg = DeclareLaunchArgument(
-        "wheel_config_path",
-        description="Path to wheel configuration file.",
-    )
-
+    wheel_type = LaunchConfiguration("wheel_type")
+    battery_config_path = LaunchConfiguration("battery_config_path")
+    components_config_path = LaunchConfiguration("components_config_path")
     controller_config_path = LaunchConfiguration("controller_config_path")
-    declare_controller_config_path_arg = DeclareLaunchArgument(
-        "controller_config_path",
-        description="Path to controller configuration file.",
-    )
+    namespace = LaunchConfiguration("namespace")
+    publish_robot_state = LaunchConfiguration("publish_robot_state")
+    wheel_config_path = LaunchConfiguration("wheel_config_path")
+    use_sim = LaunchConfiguration("use_sim")
```

Comment:
APPLIES TO ALL

Is it worth instantiating the `LaunchConfiguration` object if used only once in the whole file? What do you think about specifying `LaunchConfiguration("<arg_name>")` in-place, and instantiating it only if used at least twice?